packages/test/src/workflows/cancel-requested-with-non-cancellable.ts

```diff
@@ -0,0 +1,20 @@
+// @@@SNIPSTART nodejs-cancel-requested-with-non-cancellable
```

Comment:
I think these workflows which are part of the docs should have some brief high-level comments describing what they illustrate / the semantics (some already do, but the rest)